src/main/java/ch/njol/skript/expressions/ExprFormatTime.java

```diff
@@ -40,7 +40,7 @@
 import ch.njol.util.Kleenean;
 
 @Name("Formatted time")
-@Description("Converts date to human-readable text format. By default, yyyy-MM-dd HH:mm:ss z will be used. For reference, see this "
+@Description("Converts date to human-readable text format. By default, yyyy-MM-dd HH:mm:ss (e.g. '2018-03-30 16:03:12') will be used. For reference, see this "
```

Comment:
Are you sure the `z` wasn't a part of the format? Isn't it the timezone or something?

Reply:
Yeah, but in this case it isn't really necessary, since 0930 (09:30) is already the defined timezone (according to the wikipedia article, and yes i actually read it)

Reply:
Where is that `0930 (09:30)` and how does it define the timezone

Reply:
`"Z" is the zone designator for the zero UTC offset. "09:30 UTC" is therefore represented as "09:30Z" or "0930Z". Likewise, "14:45:15 UTC" is written as "14:45:15Z" or "144515Z"` - Wikipedia https://en.wikipedia.org/wiki/Time_zone

Reply:
It says you can use 16:03:12Z to define that it is UTC, not just remove it. But this is different, see https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html.

The default format has `z` in it which will be replaced with UTC, PST etc.